packages/core/admin/admin/src/content-manager/pages/App/index.js

```diff
@@ -73,6 +72,9 @@ const App = () => {
   const { formatMessage } = useIntl();
   const { startSection } = useGuidedTour();
   const startSectionRef = useRef(startSection);
+  const permissions = useSelector(selectAdminPermissions);
+
+  console.log(permissions);
```

Comment:
wanna flag this now fyi

Reply:
Please leave me alone in here until my tears have dried.